testsuite/features/step_definitions/navigation_steps.rb

```diff
@@ -461,6 +466,14 @@
   end
 end
 
+Then(/^the table row for "([^"]*)" should( not)? contain "([^"]*)" icon$/) do |row, should_not, icon_class|
```

Comment:
Personally I would prefer to hide internal details like the class name from the .feature file

Reply:
Ok, that makes sense. I enhanced the step definition (`if-else` for now, since there is just one icon supported. If this gets used in the future, it can be rewritten using `case` maybe)...